public/.htaccess

```diff
@@ -20,7 +20,8 @@ Options +SymLinksIfOwnerMatch
     RewriteRule !^admin index.php?%{QUERY_STRING} [L]
 </ifModule>
 
-<IfModule mod_php7.c>
-  php_value upload_max_filesize 50M
-  php_value post_max_size 100M
-</IfModule>
\ No newline at end of file
+# Uncomment the following lines to midify the PHP variables like increase the max upload size of Directus
```

Comment:
Haha I'm fine with ignoring my suggestion, but here's another one to fix the spelling/grammar error 😉 

```suggestion
# Uncomment the following lines to modify PHP settings. The lines below can be used to increase the max upload size of files in Directus
```

Reply:
@rijkvanzanten I am not ignoring your suggestions. I am really sorry if you feel that way.
 I want to make sure that comment shows, *Anyone can modify any available PHP variables from `.htaccess`*

Thanks a lot for correcting me. 🙂

Reply:
Haha I'm just playing with ya. No hard feelings 😘